test/device/wavefront.jl

```diff
@@ -1,4 +1,11 @@
 @testset "Wavefront Operations" begin
+    wavefrontsize = Ref{UInt32}()
+    AMDGPU.HSA.LibHSARuntime.hsa_agent_get_info(
+        AMDGPU.default_device().agent,
+        AMDGPU.HSA.AGENT_INFO_WAVEFRONT_SIZE,
+        wavefrontsize
+    )
```

Comment:
Would you mind adding a method `getinfo(::ROCDevice, ..., ...)` in `src/device.jl` that allows us to pass `AMDGPU.default_device()`? And then this can be:
```julia
wavefrontsize = Ref{UInt32}()
AMDGPU.Runtime.getinfo(AMDGPU.default_device(), AMDGPU.HSA.AGENT_INFO_WAVEFRONT_SIZE, wavefrontsize)
```

We could also add a `wavefrontsize(::ROCDevice)` function which does this for us, which would be really convenient.